code/datums/tutorial/xenomorph/abomination.dm

```diff
@@ -0,0 +1,214 @@
+/datum/tutorial/xenomorph/abomination
+	name = "Xenomorph - Abomination"
```

Comment:
It would be ideal if this was sorted after Xenomorph - Basic in the list, but I assume its currently listed before because A < B

Reply:
done